src/main/kotlin/no/nav/syfo/domain/AktivitetskravVurdering.kt

```diff
@@ -39,21 +36,147 @@ sealed class VurderingArsak(val value: String) {
     }
 }
 
-data class AktivitetskravVurdering(
-    val uuid: UUID,
-    val createdAt: OffsetDateTime,
-    val createdBy: String,
+sealed class AktivitetskravVurdering(
+    open val uuid: UUID,
+    open val createdAt: OffsetDateTime,
+    open val createdBy: String,
     val status: AktivitetskravStatus,
-    val arsaker: List<VurderingArsak>,
-    val beskrivelse: String?,
-    val stansFom: LocalDate?,
-    val frist: LocalDate?,
-    val varsel: AktivitetskravVarsel?,
+    val isFinal: Boolean,
 ) {
 
-    fun isFinal() = this.status.isFinal
+    data class Avvent(
+        override val uuid: UUID,
+        override val createdAt: OffsetDateTime,
+        override val createdBy: String,
+        val arsaker: List<Arsak>,
+        val beskrivelse: String,
+        val frist: LocalDate?,
+    ) : AktivitetskravVurdering(uuid, createdAt, createdBy, status = AktivitetskravStatus.AVVENT, isFinal = false) {
+        enum class Arsak(val value: String) {
+            OppfolgingsplanArbeidsgiver("OPPFOLGINGSPLAN_ARBEIDSGIVER"),
```

Comment:
Må disse mulig verdiene her nå være i synk med `VurderingArsak`-klassene over her eller er det mulig å rydde opp i senere?

Reply:
Tenkte å rydde opp i `VurderingArsak` i en annen PR 😊 Det er brukt veldig mange steder, så kan bli en del endringer for å bli kvitt det